packages/ui/src/council/components/PastVotes/PastVote/PastVote.tsx

```diff
@@ -36,7 +38,10 @@ export const PastVote = ({ vote, $colLayout }: PastVoteProps) => {
     })
   }
 
-  const isDisabled = !vote.stakeLocked || isTransactionPending
+  const isVoteStakeLocked = useIsVoteStakeLocked(vote.voteFor, false)
```

Comment:
The election should be finished here
```suggestion
  const isVoteStakeLocked = useIsVoteStakeLocked(vote.voteFor, true)
```